libraries/Microsoft.Bot.Builder.Dialogs/DialogSet.cs

```diff
@@ -41,6 +43,7 @@ public DialogSet(IStatePropertyAccessor<DialogState> dialogState)
         public DialogSet()

```

Comment:
Can we make this constructor call the other constructor that takes parameters with the default values to avoid duplicating code? 

Also, one constructor throws and exception if dialogState is not specified and the other one doesn't do we know why is that? will DialogSet work if dialotState is null

Reply:
I do not know why there is different behavior between the two constructors, but this differing behavior is why the one does not call the other. It seems there are instances where the default constructor is used (see DialogContainer), and the internal dialog state property accessor appears to be unused.

Reply:
I created an issue to investigate this and [eliminate the empty constructor](https://github.com/microsoft/botbuilder-dotnet/issues/6138) if possible once we cleanup the other code from dialogs.